python/src/wrfcloud/jobs/job.py

```diff
@@ -134,6 +150,266 @@ def sanitized_data(self) -> Union[dict, None]:
             return None
         return data
 
+    @property
+    def wps_code_dir(self) -> str:
+        """
+        Get the directory with WPS code
+        """
+        return os.environ['WPS_HOME'] if 'WPS_HOME' in os.environ else '/home/ec2-user/WPS'
```

Comment:
This could be simplified:
```suggestion
        return os.environ.get('WPS_HOME', '/home/ec2-user/WPS')
```